blocks/extensions/saveload.py

```diff
@@ -26,6 +26,12 @@ class SerializeMainLoop(SimpleExtension):
     ----------
     path : str
         The destination path for pickling.
+    save_separately : list of str, optional
+        The list of the main loop's attributes to be pickled separately
+        to their own files. The pathes will be formed by adding
```

Comment:
*paths

Would it be a good idea to set sensible defaults for this i.e. didn't we say that the log and the dataset iterator should probably be saved separately>